src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, Countable {

```

Comment:
The class `Figure` should not implement interfaces that contain unrelated methods. The `draw()` and `getArea()` methods are conceptually different and should be in separate interfaces.